app/src/configs/env.ts

```diff
@@ -58,6 +58,11 @@ export const DISABLE_LOGIN_PROTECTION = process.env.REACT_APP_DISABLE_LOGIN_PROT
 /** Activate the product-catalogue menu */
 export const ENABLE_PRODUCT_CATALOGUE = process.env.REACT_APP_ENABLE_PRODUCT_CATALOGUE === 'true';
 
+/** Activate teams menu */
+export const ENABLE_TEAMS_MODULE = process.env.REACT_APP_ENABLE_TEAMS_MODULE === 'true';
```

Comment:
shouldn't this be part of teams module ?

Reply:
@ashfahan It seems it was removed from master, Let me remove it